application/views/admin/survey/Question/massive_actions/_set_questions_other.php

```diff
@@ -3,11 +3,15 @@
  * Set question group and position modal body (parsed to massive action widget)
  */
 ?>
-<form class="custom-modal-datas form-horizontal">
+<form class="custom-modal-datas form-horizontal" data-trigger-validation="true">
     <div  class="form-group" id="OtherSelection">
         <label class="col-sm-4 control-label"><?php eT("Option 'Other':"); ?></label>
         <div class="col-sm-8">
-            <?php $this->widget('yiiwheels.widgets.switch.WhSwitch', array('name' => 'other', 'value'=> '', 'htmlOptions'=>array('class'=>'custom-data  bootstrap-switch-boolean', 'data-gridid'=>'question-grid'), 'onLabel'=>gT('On'),'offLabel'=>gT('Off')));?>
+            <select class="form-control custom-data attributes-to-update" id="other" name="other" required>
+                <option value="" selected="selected"><?php eT('Please select and option');?></option>
```

Comment:
Please select **and** option ?

Please fix this text (check in the current master for the wording) and then merge please.

Reply:
> (check in the current master branch for the new wording)

Master looks good.
https://github.com/LimeSurvey/LimeSurvey/blob/master/application/views/admin/survey/Question/massive_actions/_set_questions_other.php#L15